pkg/operator/cluster/ceph/osd/osd.go

```diff
@@ -53,6 +52,7 @@ const (
 	OrchestrationStatusFailed        = "failed"
 	appName                          = "rook-ceph-osd"
 	appNameFmt                       = "rook-ceph-osd-%s"
+	osdPodNameFmt                    = "rook-ceph-osd-cluster-%s-node-%s-id-%s" // cluster-node-id
```

Comment:
what about this simplified pod name: `rook-ceph-osd-%s-%s`? where the first parameter is the node name and second parameter is osd id. As long as we have one cluster per namespace, we don't need to include the cluster name.